From 335c5925d5400c9c5222aebbf3f33a941ab1a8a2 Mon Sep 17 00:00:00 2001
From: Andreas Mohr <and@gmx.li>
Date: Tue, 19 Jan 2016 17:00:24 +0000
Subject: [PATCH] fix wrong lowercase sed syntax

fix bug #3586 reported by rimf
(plus minor tweak remove unneeded cat/pipe command)

Signed-off-by: Andreas Mohr <and@gmx.li>
---
 misc/mcedit.menu.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/misc/mcedit.menu.in b/misc/mcedit.menu.in
index e64e55f..e1aa510 100644
--- a/misc/mcedit.menu.in
+++ b/misc/mcedit.menu.in
@@ -471,13 +471,13 @@ o       Open bash to next free console
 u       Upper case selection
         TMPFILE=`mktemp ${MC_TMPDIR:-/tmp}/up.XXXXXX` || exit 1
         cat %b > $TMPFILE
-        cat $TMPFILE| sed 's/\(.*\)/\U\1/' >%b
+        sed 's/\(.*\)/\U\1/' $TMPFILE >%b
         rm -f $TMPFILE
 
 l       Lower case selection
         TMPFILE=`mktemp ${MC_TMPDIR:-/tmp}/up.XXXXXX` || exit 1
         cat %b > $TMPFILE
-        cat $TMPFILE| sed 's/\(.*\)/\U\1/' >%b
+        sed 's/\(.*\)/\L\1/' $TMPFILE >%b
         rm -f $TMPFILE
 
 #-------------------------- End of common section -----------------------------
